### ISSUE TITLE
Link Back To Airmail from Todoist Entry Not Active

### ISSUE THREAD

**Issue by Seviant** [Airmail/Todoist-AMPlug#1]
When creating a Todoist entry from the Airmail application, the only data that is populated is the subject line of the email. The body of the email is not supported, nor are any replies, dates, recipients or attachments. The ONLY thing that gets placed in the Todoist task is the subject line of the email from Airmail. 

The "live" link created in the Todoist Task that should link back to the email is dead. For example, the link will look similar to the following: "airmail://message?mail=seviant%40seviant.org&messageid=aeb9e1e7-8986-4ce6-b8c6-0479c480d83e%40xtinmta4114.xt.local" however the user does not have the ability to click on the link and has no easy way of clicking back to the email contained in the Airmail Desktop application.

Please see screenshot attached.
![screen shot 2015-03-31 at 3 59 01 pm](https://cloud.githubusercontent.com/assets/7832192/6931166/edf1538c-d7be-11e4-8916-f2ab51c74e13.png)

**Comment by bkleef** [comment]
Any ideas what can be done to make the URL clickable? It's basically useless without being clickable.

**Comment by Seviant** [comment]
No. I still have not received a response to this problem. Moved on to using Evernote+Podio as a task manager.
